source/release-notes/1.6.txt

```diff
@@ -42,6 +42,50 @@ For more information, see :ref:`c2c-filter-regex`.
 
 Starting in 1.6.0, ``mongosync`` supports sync with MongoDB 7.0.
 
+Other Notes
+~~~~~~~~~~~
+
+New Features:
+
+- Sped up the commit process by parallelizing metadata cleanup. 
+  (:issue:`REP-2568`)
+
+- Reduced memory allocation during change event application by ~30% for 
+  pre-6.0 releases of MongoDB Server versions. (:issue:`REP-3015`)
+
+- Changed telemetry to also send error messages to Segment when ``mongosync``
+  exits with a fatal error. (:issue:`REP-3138`) 
+
+- Added support for ``mongosync`` to sync between source and destination 
+  clusters that have the same major and minor versions, but different 
+  patch versions. (:issue:`REP-3161`)
+
+- Upgraded build to use Go 1.20. (:issue:`REP-3044`)
+
+- ``mongosync`` now periodically logs when server operations take a longer 
+  than expected time. (:issue:`REP-3176`)
+
+Issues Fixed:
+
+- Fixed bug introduced in 1.5, where ``mongosync`` could fail to apply change
```

Comment:
[nit]

We should be consistent in these bullets as to whether we are saying "Fixed bug" or "Fixed a bug".

Reply:
Fixed